### DISCUSSION TITLE
Specific Text Font when content is published.

### DISCUSSION THREAD

**Discussion by Polo224** [community/community#105151] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi All, 

Please forgive me because I am a newbie and I really have no idea how this all works. I am trying to update a project created by someone else and I have been asked to 'change' the font displayed online once the content is published. I don't know if there is a way to select a specific font or if we have to stick to the standard look we get when we publish. Does any of you know how to help me with this? The font I am trying to apply is Arial Black.

Thanks in advance to you all for the help.
Polo :)

**Comment by gil-son** [comment]
Hello,

Please share the link of that project and if possible give more details

**Reply by Polo224** [reply]
Hi Gil-son, 

Thanks for getting back to me. Below you can find the link of one of the pages I am working on. Is this what you need? 

https://github.com/ConquestSolutions/conquestsolutions.github.io/edit/master/versioned_docs/version-Conquest%204/11-Getting%20Around%20Conquest/06-Navigating%20in%20Conquest%204/41-forms.md


Details: This md page, once published will be part of a Help page for a website (https://help.conquestsoftware.com.au/forms). If you open the help page you'll see that the font on the page is not Arial Black. What I am asking is if there is a way for me to code the md page so that when I save the changes the help desk will show the Arial Black font. Is this something I can do in github or I should use a different software like Docusaurus for example?

**Reply by gil-son** [reply]
No need to worry, Polo! I'm here to help you. Changing the font in a web project is typically done using CSS (Cascading Style Sheets). If you want to use Arial Black, you can specify it in your CSS. Here's a simple guide:

- Locate Your CSS File:
        In your project, there should be a CSS file. It might be named something like styles.css, or it could be part of an HTML file within a <style> tag.

- Add Font Family:
 Find the selector for the element you want to change the font for. It might be something like body, p, h1, or have a specific class or ID. Once you've identified the selector, add the font-family property and set it to "Arial Black". For example: css

```
body {
  font-family: 'Arial Black', sans-serif;
}
```

If make sense, consider to mark the best answer

**Reply by Polo224** [reply]
Thanks Gil-son, it totally make sense but I might still need help. I found 2 separate css files but I can't seem to understand which is the correct one and where to add the font property. I have added a couple of screenshots :)

![Screenshot 2024-02-05 142008](https://github.com/community/community/assets/158120072/d06d3a8f-6048-4e57-9a60-9ac0c49bc824)
![Screenshot 2024-02-05 142022](https://github.com/community/community/assets/158120072/ee844c80-5b9b-4145-88b9-1f18772ebc6d)


Thanks again for your help

**Reply by gil-son** [reply, marked as answer]
It's my pleasure. Please consider make it the best answer. God bless you!
